cypress/integration/basic.spec.js

```diff
@@ -0,0 +1,10 @@
+import { tools } from "../support/constants.js";
+
+describe("Drawing elements", () => {
+  it("should draw rectangle", () => {
+    cy.visit("/");
+    cy.viewport(600, 600);
+    cy.pickTool(tools.rectangle);
+    cy.get("#canvas").canvasDrag({ x: 50, y: 50 }, { x: 100, y: 100 });
```

Comment:
This looks super easy to write test in this way! Can’t wait to start using it :)

Reply:
We'll see how assertions turn out. Right now, this does nothing except draws things :)